libraries/rush-lib/src/cli/actions/ScanAction.ts

```diff
@@ -56,14 +61,31 @@ export class ScanAction extends BaseConfiglessRushAction {
       parameterLongName: '--all',
       description: 'If this flag is specified, output will list all detected dependencies.'
     });
+    this._folders = this.defineStringListParameter({
+      parameterLongName: '--folder',
+      parameterShortName: '-f',
+      argumentName: 'FOLDER',
+      description:
+        'The folders that need to be scanned, default is src and lib.' +
+        'Normally we can input all the folders under the project directory, excluding the ignored folders.'
+    });
+    this._projects = this.defineStringListParameter({
+      parameterLongName: '--only',
+      parameterShortName: '-o',
+      argumentName: 'PROJECT',
+      description: 'Projects that need to be checked for phantom dependencies.'
+    });
```

Comment:
Ideally this would support all project selection parameters. There is a way to do that for an arbitrary action.

Reply:
yeah, but i feel that this place is not very suitable for other selection capabilities, such as --to, because some projects may have their own directory structure. If it is --to, it will scan the same folder for these projects.

Reply:
In actual use, I will detect which folders under the project will be submitted, and then pass these folders to the scan command one by one for scanning, so I think using the --project flag is sufficient.